src/scripts/push-image.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+ORB_VAL_ACCOUNT_URL="${ORB_ENV_REGISTRY_ID}.dkr.ecr.${ORB_EVAL_REGION}.amazonaws.com"
```

Comment:
Region is not being evaluated here. Env registry ID also is not passed to the environment + is not being indirectly called

Reply:
Changed it to this: `ORB_VAL_ACCOUNT_URL=$(eval echo "${ORB_ENV_REGISTRY_ID}.dkr.ecr.${ORB_EVAL_REGION}.amazonaws.com")`

Reply:
I used `eval` on the region and then expanded the registry id like so:
`ORB_VAL_ACCOUNT_URL="${!ORB_ENV_REGISTRY_ID}.dkr.ecr.${ORB_EVAL_REGION}.amazonaws.com"`